tokio/src/fs/file.rs

```diff
@@ -837,6 +854,19 @@ mod preadv2 {
         }
     }
 
+    #[cfg_attr(test, allow(unused))]
+    pub(crate) fn has_buggy_preadv2() -> bool {
```

Comment:
This should probably link to the relevant bug. Probably [this email thread](https://lore.kernel.org/linux-fsdevel/fea8b16d-5a69-40f9-b123-e84dcd6e8f2e@www.fastmail.com/T/#u)?